release_tools/semverup.py

```diff
@@ -171,20 +171,29 @@ def read_version_number(filepath):
 def force_new_version_number(current_version, bump_version, pre_release):
     """Increment version number based on bump_version choice"""
 
-    if current_version.prerelease and not pre_release:
-        return current_version.finalize_version()
-    elif current_version.prerelease and pre_release:
-        return current_version.bump_prerelease()
+    next_version = None
 
-    if bump_version == 'MAJOR':
-        next_version = current_version.bump_major()
-    elif bump_version == 'MINOR':
-        next_version = current_version.bump_minor()
-    elif bump_version == 'PATCH':
+    if bump_version == 'PATCH' and not current_version.prerelease:
         next_version = current_version.bump_patch()
+    if bump_version == 'MINOR':
```

Comment:
This functionality is unrelated the the feature described on the PR. Please put all this code in a different PR.